src/message/install-certificate/index.tsx

```diff
@@ -93,10 +100,11 @@ export const InstallCertificate: React.FC<InstallCertificateProps> = ({
   };
 
   const installCertificateData = new InstallCertificateData();
-  const installCertificate = new Certificate();
-  installCertificate[CertificateProps.id] = NEW_IDENTIFIER as unknown as number;
-  installCertificateData[InstallCertificateDataProps.certificate] =
-    installCertificate;
+  // const installCertificate = new Certificate();
+  // installCertificate[CertificateProps.id] = NEW_IDENTIFIER as unknown as number;
+  // installCertificateData[InstallCertificateDataProps.certificate] =
+  //   installCertificate;
+  installCertificateData[InstallCertificateDataProps.certificate] = NEW_IDENTIFIER;
```

Comment:
You don't need this right? We use NEW_IDENTIFIER to show the "Select" button, which signifies that the record is in new / create state where the associated id has not yet been set. Since you are switching this to just be a string, you should not need this line I believe 🤔